src/Utilities.java

```diff
@@ -0,0 +1,38 @@
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import java.util.Scanner;
+
+public class Utilities {
+
+    public static String readFromUser(){
+        System.out.println("Type your season:");
+        Scanner scanner = new Scanner(System.in);
+        return scanner.nextLine();
+    }
+
+    public static void showSeasonMonths(Season season){
+        System.out.println("Months that are assigned to that season are:");
+        System.out.println(Arrays.toString(season.getMonths()));
+    }
+
+    public static Optional<Season> getSeason(String userInput) {
+        Season[] seasons = Season.getValues();
+        for (Season season : seasons) {
+            if (season.getDescription().equalsIgnoreCase(userInput))
+                return Optional.of(season);
+        }
+        return Optional.empty();
+    }
+
+    public static void listMonthsForSeason(Optional<Season> season){
```

Comment:
odpowiadajac na pytanie z komentarza : Optionala powinnismy korzystac tylko przy zwracaniu metody zeby wyrazic ze rezultat moze nie zostax zwrocony. Bardzo dobrze opisane jest to tu:

http://dolszewski.com/java/java-8-optional-use-cases/